src/api/lib/middlewares/tokenChecker.js

```diff
@@ -17,6 +17,11 @@ export default async (ctx, next) => {
 
     try {
         ctx.user = await jwt.verify(token, config.apps.api.security.jwt.privateKey);
```

Comment:
Use `ctx.payload` instead or `ctx.tokenPayload` or `ctx.tokenData`, because it is not only `user`.